src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+  background-image: url(images/logo.png);
+}
+
+.header__nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  padding-right: 20px;
+}
+
+.nav__link {
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  display: block;
+  padding: 23px 0;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: #000;
+}
+
+.nav__item:last-child {
+  padding-right: 0;
+}
+
+.nav__item:first-child {
+  padding-left: 0;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+a[data-qa="hover"]:hover {
```

Comment:
why do you style it using tag and attribute?
just add some additional class if you need